pkgs/top-level/all-packages.nix

```diff
@@ -18255,6 +18255,8 @@ with pkgs;
 
   cups-bjnp = callPackage ../misc/cups/drivers/cups-bjnp { };
 
+  cnijfilter = callPackage_i686 ../misc/cups/drivers/cnijfilter { };
```

Comment:
Per your comment about versioning, we should probably call this `cnijfilter_2_80` so it won't conflict with  other versions of the driver.